doozer/doozerlib/distgit.py

```diff
@@ -1672,48 +1679,31 @@ def _determine_upstream_rhel_version(self, source_path) -> Optional[int]:
 
         # We will infer the rhel version from the last build layer in the upstream Dockerfile
         last_layer_pullspec = parent_images[-1]
-        redis_key = f'appdata:upstream-el-version:{last_layer_pullspec}'
-
-        # If the pullspec is not tracked in Redis yet, this will return None
-        version = redis.get_value_sync(redis_key)
-        if version:
-            # Already tracked in Redis
-            return int(version)
 
-        # Not tracked yet: let's read it from os-release
+        version = None
         try:
-            # Create a temp dir with a random name to store the os-release file
-            with TemporaryDirectory() as tmpdir:
```

Comment:
`TemporaryDirectory` imported symbol is no longer used

Reply:
Removed the import.